pyqtgraph/parametertree/parameterTypes/basetypes.py

```diff
@@ -1,11 +1,10 @@
 import builtins
+from functools import  partial
 
-from ... import functions as fn
-from ... import icons
-from ...Qt import QtCore, QtGui, QtWidgets
 from ..Parameter import Parameter
 from ..ParameterItem import ParameterItem
-
+from ... import functions as fn, icons
```

Comment:
I am sure somebody strong has opinions on how many lines this should use.  :)
I don't, but I find that this reads a little curiously when glancing over it and see "as fn, icons"

Reply:
> I am sure somebody strong has opinions on how many lines this should use. :)

I certainly don't.

I can see by some of the other changes in this commit that I must have accidentally run the code cleanup on this file before committing. It will automatically organize imports, remove trailing white space, etc. 

I've changed it back to 
```
from ... import functions as fn
from ... import icons
```